kolibri/deployment/default/settings/base.py

```diff
@@ -39,6 +39,8 @@
     os.path.join(KOLIBRI_MODULE_PATH, "locale"),
 ]
 
+OPTIONS = options.get_options_from_env()
```

Comment:
I think all of this logic should be concealed in `kolibri.utils.conf` -- because this module is expected to be loaded before the Django settings stack and can therefore define settings in one single place.

So for instance:

```
from kolibri.utils import conf

DATABASES = conf.global_settings.DATABASES
```

Another thing: `conf.global_settings` can be made a completely read-only instance after instantiating to guarantee against wrong use.

A friend of mine just tipped me that even for rigorous cases, they would replace `os.environ` with a read-only dict. It's maybe not nice to do against other external environments, but for testing purposes it could be interesting to avoid any changes beyond loading the environment in `kolibri.utils.conf`

Reply:
>I think all of this logic should be concealed in kolibri.utils.conf

I avoided using that module itself because it's cluttered with a lot of top-level stuff very specific to parsing `kolibri_settings.json`, so putting this new logic in there would be really messy.

>avoid any changes beyond loading the environment in kolibri.utils.conf

I'm almost done with a new version of this PR that doesn't modify `os.environ` at all (just looks to it for overrides -- everything will be funneled through the strict conf file config as you suggested). Stay tuned.